coverage-reports/pom.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
+
+  Copyright (C) 2020-2022 Huawei Technologies Co., Ltd. All rights reserved.
+
   ~ Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
6,7,8 lines are for the ASF which should not be used in this file.
Please remove them all.
The other source codes have the same issue.